mscore/pianolevelsfilter.h

```diff
@@ -55,8 +56,10 @@ class PianoLevelsFilter
 
 class PianoLevelFilterOnTime : public PianoLevelsFilter
 {
+      Q_DECLARE_TR_FUNCTIONS(PianoLevelFilterOnTime)
+
 public:
-      QString name() override { return "On Time"; }
+      QString name() override { return tr("On Time"); }
```

Comment:
Good catch! But since the term "On Time" can be ambiguous without explanation, I suggest you add a translator comment to it. You can look into `menus.cpp` and search "swing" in there, and you'll see an example.

And actually, is it really necessary to have the first letter of all words capitalized? It seems to me there's nothing special with these strings that needs emphasizing them like this. Just write them as normal strings.